styleguide/components/11-base/boxes/_boxes.scss

```diff
@@ -72,10 +72,10 @@
 
 .box-top {
   .inner-box {
-    margin-bottom: 1.5rem;
```

Comment:
Have you checked if the removal of this `margin-bottom` doesn't break anything in any layouts?

Reply:
Hmz, after a double check... it breaks the accolade in summary box.

But the point is: none of the other boxes have inner-box bottom margin.
And it breaks our rule that the context is responsible for the margin of components.

Suggest we move it to .summary-box, but I'm not sure how that matches with our planned refactoring.

Reply:
Moving it to summary-box sounds fine to me. 
(I checked the [refactoring](https://digipolisgent.atlassian.net/wiki/spaces/UXWEB/pages/1101726698/Analyse+Stijlgids+componenten); turns out this component will be preserved)

Reply:
Allright moved it.
I guess it would take us too far to rename summary-box to accolade-box and make it just another box variant.

Reply:
That's undoubtedly going to happen, but indeed beyond the scope of this PR. 👍